catalog/spatial/registry/registry-common/src/main/java/org/codice/ddf/registry/common/metacard/RegistryObjectMetacardType.java

```diff
@@ -98,39 +98,29 @@ public RegistryObjectMetacardType(String name, Set<AttributeDescriptor> descript
     }
 
     private void addRegistryAttributes() {
```

Comment:
Please see https://github.com/codice/ddf/pull/1084/commits/6fac4b26c45cd92b8e7a9ae89886d28a9b0a453f
for the changes minus re-ordering.